JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -193,44 +193,47 @@ int main(int argc,char**argv)
         continue;
       }
 
-      //if(verbose>0) cout << "Attempting to fit " << histname << " ... " << endl;
-
-      //if (histname.find("RelRsp")>5&&histname.find("AbsRsp")!=0) {
-      //  hrsp->Write();
-      //  continue;
-      //}
-      
-      if (hrsp->Integral()>0.0) {
+      //
+      // write all the histogram used later for tau calibration study
+      //
+      if( fittype==2 && (histname.find("RelRsp_JetEta")==string::npos || histname.find("_RefPt")==string::npos) ) {
```

Comment:
I tried this some time ago, but I got troubles, for me at this point it is more simple.